packages/slate-react/src/components/content.js

```diff
@@ -206,7 +207,7 @@ class Content extends React.Component {
     native.removeAllRanges()
 
     // COMPAT: IE 11 does not support Selection.extend
-    if (native.extend) {
+    if ((native.extend && !IS_IOS) && (native.extend && !IS_SAFARI)) {
```

Comment:
Maybe ``if (native.extend && !(IS_SAFARI || IS_IOS))`` for more clarity?

Reply:
you are indeed correct! ;)